### DISCUSSION TITLE
Unable to launch on M1 Max Macbook Pro due to OBS not being able to create the required files

### DISCUSSION THREAD

**Discussion by newdev4ios** [obsproject/obs-studio#8291] [Development Questions] [answered]
Unable to Launch OBS at all due to the program not being able to create any required files apparently, I had to manually create the folders it was requesting but now I'm stuck at Failed to open global.ini: -1 then closing that error pops up Failed to initialize global config

Help, this is probably not the only file its not able to find/create, once this file is created or modified then it'll probably ask for more. Any Ideas as to what to do to fix this please

**Comment by sebastian-s-beckmann** [comment, marked as answer]
This appears to be a support request, for which we don't use GitHub Discussions.
Please make sure to visit the support sections on our [Discord server](https://discord.gg/obsproject) or [Forums](https://obsproject.com/forum).

Thank you!
